schema/xml/metaschema.xsd

```diff
@@ -960,7 +961,7 @@
       <xs:extension base="ExpectConstraintType">
         <xs:attribute name="target" type="MetaschemaPathType" use="required">
           <xs:annotation>
-            <xs:documentation>Specifies the target of the constraint as a metaschema path. If the value is "." and the containing metaschema object is a field or flag, the constraint applies to the value of the field or flag. Otherwise, the scope value "." is not allowed to be used.</xs:documentation>
+            <xs:documentation>Specifies the target of the constraint as a Metaschema path. If the value is "." and the containing Metaschema object is a field or flag, the constraint applies to the value of the field or flag. Otherwise, the scope value "." is not allowed to be used.</xs:documentation>
```

Comment:
```suggestion
            <xs:documentation>Specifies the target of the constraint as a Metaschema expression. If the value is "." and the containing Metaschema object is a field or flag, the constraint applies to the value of the field or flag. Otherwise, the scope value "." is not allowed to be used.</xs:documentation>
```